aiida/engine/utils.py

```diff
@@ -26,6 +30,30 @@
 PROCESS_STATE_CHANGE_KEY = 'process|state_change|{}'
 PROCESS_STATE_CHANGE_DESCRIPTION = 'The last time a process of type {}, changed state'
 
+LAUNCHER_SIGNATURE_DEPRECATION = """\
```

Comment:
I'd remove this continuation `\` here:

```suggestion
LAUNCHER_SIGNATURE_DEPRECATION = """
```
So the start of the message is on a new line, I find it more readable:

![Screenshot 2023-12-08 at 11 00 41](https://github.com/aiidateam/aiida-core/assets/25607960/03fc5adf-539e-41e6-b96d-3e725ad6280a)

Reply:
Done